peripherals/usart/_v2_BC_common.yaml

```diff
@@ -0,0 +1,86 @@
+# Common fields between v2 B and C peripherals
+
+CR1:
+  OVER8:
+    Oversampling_16: [0, "Oversampling by 16"]
+    Oversampling_8: [1, "Oversampling by 8"]
+  EOBIE:
+    Inhibited: [0, "Interrupt is inhibited"]
+    Interrupted: [1, "A USART interrupt is generated when the EOBF flag is set in the ISR register"]
```

Comment:
I think these two should be `Enabled` and `Disabled` to match other `*IE` registers like `TXEIE`?